lib/rubycritic/cli/options.rb

```diff
@@ -39,6 +39,10 @@ def parse
             @suppress_ratings = true
           end
 
+          opts.on("-O ", "--open", "Open report with browser") do |open_with|
```

Comment:
Let's add all the possibilities here a user can enter, otherwise it's just happy guessing ;)